core/src/main/resources/org/apache/spark/ui/static/webui.js

```diff
@@ -87,4 +87,11 @@ $(function() {
   collapseTablePageLoad('collapse-aggregated-runningExecutions','aggregated-runningExecutions');
   collapseTablePageLoad('collapse-aggregated-completedExecutions','aggregated-completedExecutions');
   collapseTablePageLoad('collapse-aggregated-failedExecutions','aggregated-failedExecutions');
-});
\ No newline at end of file
+});
+
+$(function() {
+    // Trigger a double click on the span to show full job description.
+    $(".description-input").dblclick(function() {
+        $(this).removeClass("description-input").addClass("description-input-full");
+    });
+});
```

Comment:
We supported this feature before. https://github.com/apache/spark/pull/25374/files#diff-c794386db0ef43c8090b67c83cf6b950L81-L89

Reply:
cc @gengliangwang @sarutak

Reply:
@wangyum yes. It was present. But 
https://github.com/apache/spark/pull/25374/files
removed additonal-metrics.js whch had the function for this double click event.
So this feature was broken.

Reply:
Could you update your change to
```javascript
$(function() {
    // Show/hide full job description on click event.	
    $(".description-input").click(function() {	
        $(this).toggleClass("description-input-full");	
    });
});
```

Reply:
@wangyum i have changed. could you check